aws-amp/adot/README.md

```diff
@@ -0,0 +1,123 @@
+# Kubecost with AWS Distro for Open Telemetry
+
+## Overview
+
+This guide will walk you through the steps to deploy Kubecost with AWS Distro for Open Telemetry (ADOT) to collect metrics from your Kubernetes cluster.
```

Comment:
Is it worth adding some information about why someone may want to use ADOT as opposed to the other options?

Reply:
Hard +1. I'd like to know what the use cases for this are. Assume the reader doesn't know what ADOT or AMP is.

Reply:
I don't want to push one method over another.